docs/install.md

```diff
@@ -367,7 +368,7 @@ Events:
   Normal  NewDeployment  58s   pmem-csi-operator  Processing new driver deployment
   Normal  Running        39s   pmem-csi-operator  Driver deployment successful
 
-$ kubectl get po
+$ kubectl get po -n pmem-csi
```

Comment:
While at it, can you replace `po` with `pods`?

Reply:
Done.